source/tests/pt/test_finetune.py

```diff
@@ -109,6 +110,21 @@ def tearDown(self) -> None:
         FinetuneTest.tearDown(self)
 
 
+class TestEnergyZBLModelSeA(unittest.TestCase, FinetuneTest):
+    def setUp(self):
```

Comment:
need to fix this.

Reply:
Yes, we may have to consider the case that the type maps of different model are distinct..